<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kakaoDia</class>
 <widget class="QDialog" name="kakaoDia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KaKaO Map and RoadView API</string>
  </property>
  <widget class="QLabel" name="jusoTitle">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jusoSearch">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>34</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="apiCode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ApiCodeTitle">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>15</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>API Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="jusoInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>35</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>139</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QgsProjectionSelectionWidget" name="mQgsProjectionSelectionWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>61</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="coordTitle">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>66</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Set Coord</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spotClick">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>113</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pick Sp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ShowRoadView">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>112</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RoadView</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="coordinate">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>113</y>
     <width>246</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MultiJusoPath">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>89</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MultiJusoTitle">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>94</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileSel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>88</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MultiSavePath">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>89</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="MultiSavePathSel">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>88</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="MultiSave">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>91</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MultiSaveRun">
   <property name="geometry">
    <rect>
     <x>361</x>
     <y>88</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveImg">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>34</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Img Sv</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
